### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ashwinraghav**

### CODE REVIEW

**Review by hiranya911 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 ---
 
 <!-- DO NOT DELETE 
 validate_template=true
 template_path=.github/ISSUE_TEMPLATE/bug.md
 -->
 
+
 ### [READ] Step 1: Are you in the right place?
+**Thank you for submitting your issue. We are operating at reduced capacity from Dec 18 2020 to Jan 4 2020. Please expect delayed responses. For more urgent requests please reach us via our  support channels https://firebase.google.com/support**
```
Comment (hiranya911) [inline .github/ISSUE_TEMPLATE/bug.md:17]: Jan 4 2021

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -1,14 +1,20 @@
 ---
-name: ⚠️ Report a Bug
+name: "⚠️ Report a Bug"
```
Comment (paulb777) [inline .github/ISSUE_TEMPLATE/bug.md:2]: Is this change intentional?

File: .github/ISSUE_TEMPLATE/fr.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ---
-name: 💡 Feature Request
+name: "\U0001F4A1 Feature Request"
```
Comment (paulb777) [inline .github/ISSUE_TEMPLATE/fr.md:2]: Is this one?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
 ---
 
 <!-- DO NOT DELETE 
 validate_template=true
 template_path=.github/ISSUE_TEMPLATE/bug.md
 -->
 
+
 ### [READ] Step 1: Are you in the right place?
+**Thank you for submitting your issue. We are operating at reduced capacity from Dec 18 2020 to Jan 4 2021. Please expect delayed responses. For more urgent requests please reach us via our  support channels https://firebase.google.com/support**
```
Comment (paulb777) [inline .github/ISSUE_TEMPLATE/bug.md:12]: Extra space between "our" and "support"

**Review by yifanyang — APPROVED** [review]
